scripts/push-and-log-docker.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+set -e
+set -x
+
+if [ ! "$#" -ne 4 ] ; then
+    echo "usage: $0 LOG_NAME BASE_TAG HASH VERSION" >&2
+    exit 1
+fi
+
+log_name="$4"
+base_tag="$2"
+hash="$3"
+version="$4"
+
+underscore_name="$(echo -n "$log_name" | tf - _)"
+
+docker push "$base_tag-$hash"
+docker push "$base_tag-$version"
+
+mkdir -p "/tmp/artifacts"
+
+log_file="/tmp/artifacts/publish-$log_name"
+
+echo "${underscore_name}_hashed: $base_tag-$hash" > "$log_file"
```

Comment:
question: shouldn't all of these be `>>` since we are going to call this script multiple times?

Reply:
No, we're writing to a different file in each case.